receiver/redisreceiver/metric_functions.go

```diff
@@ -0,0 +1,231 @@
+package redisreceiver
+
+func uptimeInSeconds() *redisMetric {
+	return &redisMetric{
+		key:        "uptime_in_seconds",
+		units:      "seconds",
```

Comment:
Since this eventually gets assigned to `MetricDescriptor.Unit` it needs to follow conventions defined in http://unitsofmeasure.org/ucum.html

@bogdandrutu do I understand it correct that we should use "c/s" form for unit, i.e. for seconds we should use "s". Is that right?

Reply:
I kind of punted on the units for this draft, but will make sure they are consistent with the spec.

Reply:
Fixed.